.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Continuous integration
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ruby/setup-ruby@v1
```

Comment:
It'd be good to test a matrix of Ruby versions here - I'd imagine 2.7, 3.0 and 3.1 are right

Reply:
Definitely 👍🏻 I intend to do that, but right now it doesn't seem to run in `3.0` due to some out of date gems.

I was intending for this PR to simply port the CI over to GitHub Actions but keep it on Ruby 2.6.

The PR #233 would follow, in which I'll implement a matrix test (which should then hopefully pass) against 2.7, 3.0 and 3.1.

Let me know if you think there's a better approach though!

Reply:
Sure, that makes sense. Do you want to set this up as a matrix of 2.6 and 2.7 first? It'd be good to have a matrix test in on the first instance incase it reveals any problems